src/pages/moderator/rewards/create.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { Container, Group, Stack, Title } from '@mantine/core';
+
+import { createServerSideProps } from '~/server/utils/server-side-helpers';
+import { getLoginLink } from '~/utils/login-helpers';
+import { BackButton } from '~/components/BackButton/BackButton';
+import { PurchasableRewardUpsertForm } from '~/components/PurchasableRewards/PurchasableRewardUpsertForm';
+
+export default function PurchasableRewardCreate() {
+  const router = useRouter();
+  const onCreated = () => router.push(`/moderator/rewards`);
+
+  return (
+    <Container size="md">
```

Comment:
Might need the `<Meta />` header component to avoid search engines from crawling this page. Check the other mod pages as an example